lib/membrane/video_compositor/handler/callback_context/init.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Membrane.VideoCompositor.Handler.CallbackContext.Init do
```

Comment:
In the core, we resigned from having separate structs for context. Instead, we have a map and a single module with the type and description of fields common for all callbacks. All callback-specific fields are described in specs and docs of particular callbacks.

Reply:
yup, but they may have one day, and a separate struct for each callback seems like overkill anyway

Reply:
What do you think about using common `callback_context` for all callbacks with "common context", and using separate ones for callbacks with different contexts? 
Currently, we have `Membrane.VideoCompositor.Handler.CallbackContext` with common callback context and `Membrane.VideoCompositor.Handler.CallbackContext.Init` specific to `handle_init`, since fields in those are disjoint.

Reply:
My point is to:
- change `Membrane.VideoCompositor.Handler.CallbackContext` to a map
- remove `Membrane.VideoCompositor.Handler.CallbackContext.Init` and describe it in `handle_init` docs
- consider putting the `init_opts` as a separate argument to `handle_init` like it's done everywhere else, then there would be no need for the context in `handle_init`